### ISSUE TITLE
[enterprise-logs] Enterprise logs chart doesn't use latest version (v1.2)

### ISSUE THREAD

**Issue by wardbekker** [grafana/helm-charts#950]
Enterprise logs is on v1.1, while the the latest release is v1.2

https://grafana.com/docs/enterprise-logs/latest/gel-releases/#v120----november-11-2021

**Comment by ssncferreira** [comment]
Good catch. The GEL release process documentation was updated, so this step does not get forgotten: https://github.com/grafana/loki-enterprise/pull/391

**Comment by trevorwhitney** [comment]
The helm chart is currently pointing a 1.2, it was updated to do so back in November: https://github.com/grafana/helm-charts/commit/481f16129e491d97a2ec89d23e0cb167b392284e

**Comment by wardbekker** [comment]
https://github.com/grafana/helm-charts/blob/6ee711b572b2641ab2be32b079625973ae83c3de/charts/enterprise-logs/values.yaml#L22

**Comment by trevorwhitney** [comment]
@wardbekker you need to pull the latest, that commit is from November, tag is 1.3.0 on head of main.
